src/Contrib/Otlp/SpanExporterFactory.php

```diff
@@ -37,28 +37,28 @@ public function fromEnvironment(): SpanExporterInterface
      */
     private function buildTransport(): TransportInterface
     {
-        $protocol = EnvironmentVariables::has(Env::OTEL_EXPORTER_OTLP_TRACES_PROTOCOL) ?
-            EnvironmentVariables::getEnum(Env::OTEL_EXPORTER_OTLP_TRACES_PROTOCOL) :
-            EnvironmentVariables::getEnum(Env::OTEL_EXPORTER_OTLP_PROTOCOL);
+        $protocol = Configuration::has(Variables::OTEL_EXPORTER_OTLP_TRACES_PROTOCOL) ?
+            Configuration::getEnum(Variables::OTEL_EXPORTER_OTLP_TRACES_PROTOCOL) :
+            Configuration::getEnum(Variables::OTEL_EXPORTER_OTLP_PROTOCOL);
         $contentType = Protocols::contentType($protocol);
 
-        $endpoint = EnvironmentVariables::has(Env::OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
-            ? EnvironmentVariables::getString(Env::OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
-            : EnvironmentVariables::getString(Env::OTEL_EXPORTER_OTLP_ENDPOINT);
+        $endpoint = Configuration::has(Variables::OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
+            ? Configuration::getString(Variables::OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
+            : Configuration::getString(Variables::OTEL_EXPORTER_OTLP_ENDPOINT);
         if ($protocol === Protocols::GRPC) {
             $endpoint .= OtlpUtil::method(Signals::TRACE);
         } else {
             $endpoint = HttpEndpointResolver::create()->resolveToString($endpoint, Signals::TRACE);
```

Comment:
This is broken, we must use `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` as-is, [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#endpoint-urls-for-otlphttp).